keyboards/ungodly/trinity/keyboard.json

```
@@ -0,0 +1,1381 @@
+            {"matrix": [2, 8], "x": 97, "y": 51, "flags": 4},
+            {"matrix": [2, 9], "x": 110, "y": 51, "flags": 4},
+            {"matrix": [2, 10], "x": 123, "y": 51, "flags": 4},
+            {"matrix": [2, 11], "x": 136, "y": 51, "flags": 4},
+            {"matrix": [2, 12], "x": 149, "y": 51, "flags": 4},
+            {"matrix": [2, 13], "x": 162, "y": 51, "flags": 4},
+            {"matrix": [2, 14], "x": 175, "y": 51, "flags": 4},
+            {"matrix": [2, 15], "x": 201, "y": 51, "flags": 4},
+            {"matrix": [2, 16], "x": 214, "y": 51, "flags": 4},
+            {"matrix": [2, 17], "x": 227, "y": 51, "flags": 4},
+            {"matrix": [1, 17], "x": 227, "y": 31, "flags": 4},
+            {"matrix": [1, 16], "x": 214, "y": 31, "flags": 4},
+            {"matrix": [1, 15], "x": 201, "y": 31, "flags": 4},
+            {"matrix": [1, 14], "x": 169, "y": 31, "flags": 4},
+            {"matrix": [1, 14], "x": 169, "y": 31, "flags": 4},
+            {"matrix": [1, 13], "x": 169, "y": 31, "flags": 4},
+            {"matrix": [1, 12], "x": 156, "y": 31, "flags": 4},
+            {"matrix": [1, 11], "x": 143, "y": 31, "flags": 4},
+            {"matrix": [1, 10], "x": 130, "y": 31, "flags": 4},
+            {"matrix": [1, 9], "x": 117, "y": 31, "flags": 4},
+            {"matrix": [1, 8], "x": 104, "y": 31, "flags": 4},
+            {"matrix": [1, 7], "x": 91, "y": 31, "flags": 4},
+            {"matrix": [1, 6], "x": 78, "y": 31, "flags": 4},
+            {"matrix": [1, 5], "x": 65, "y": 31, "flags": 4},
+            {"matrix": [1, 4], "x": 52, "y": 31, "flags": 4},
+            {"matrix": [1, 3], "x": 39, "y": 31, "flags": 4},
+            {"matrix": [1, 2], "x": 26, "y": 31, "flags": 4},
+            {"matrix": [1, 1], "x": 13, "y": 31, "flags": 4},
+            {"matrix": [1, 0], "x": 0, "y": 31, "flags": 4},
+            {"matrix": [0, 0], "x": 0, "y": 0, "flags": 4},
+            {"matrix": [0, 2], "x": 39, "y": 0, "flags": 4},
+            {"matrix": [0, 3], "x": 52, "y": 0, "flags": 4},
+            {"matrix": [0, 4], "x": 65, "y": 0, "flags": 4},
+            {"matrix": [0, 5], "x": 84, "y": 0, "flags": 4},
+            {"matrix": [0, 7], "x": 110, "y": 0, "flags": 4},
+            {"matrix": [0, 8], "x": 123, "y": 0, "flags": 4},
+            {"matrix": [0, 9], "x": 143, "y": 0, "flags": 4},
+            {"matrix": [0, 10], "x": 156, "y": 0, "flags": 4},
+            {"matrix": [0, 11], "x": 169, "y": 0, "flags": 4},
+            {"matrix": [0, 12], "x": 182, "y": 0, "flags": 4},
+            {"matrix": [0, 13], "x": 201, "y": 0, "flags": 4},
+            {"matrix": [0, 14], "x": 214, "y": 0, "flags": 4},
+            {"matrix": [0, 15], "x": 227, "y": 0, "flags": 4},
+            {"matrix": [0, 16], "x": 240, "y": 0, "flags": 4},
+            {"matrix": [0, 17], "x": 253, "y": 0, "flags": 4}
+        ],
+        "led_count": 101,
+        "max_brightness": 150,
+        "startup_mode": "EEPROM"
```

Unsure what you're trying to do with `startup_mode`...
```suggestion
        "max_brightness": 150
```